### PR TITLE
Prepare 2.1.12

### PR DESCRIPTION
**Pull request by natemcmaster**
Part of https://github.com/aspnet/AspNetCore-Internal/issues/2593

This prepares the repo to build the next 2.1.x patch.

### CODE REVIEW

File: eng/tools/BaselineGenerator/Program.cs [hunk 1]
```diff
@@ -61,7 +61,11 @@ private async Task<int> Run()
             var sourceRepository = new SourceRepository(packageSource, providers);
             if (_update.HasValue())
             {
-                return await RunUpdateAsync(inputPath, input, sourceRepository);
+                var updateResult = await RunUpdateAsync(inputPath, input, sourceRepository);
+                if (updateResult != 0)
+                {
+                    return updateResult;
+                }
```
Comment (dougbu) [inline eng/tools/BaselineGenerator/Program.cs]: Like it 👍

**Review by dougbu — APPROVED** [review]

**Comment by dougbu** [comment]
Hmm, do we have an approved 2.1 PR in this repo that will add `<PropertyGroup Condition=" '$(VersionPrefix)' == '2.1.12' ">` to PatchConfig.props?

**Comment by natemcmaster** [comment]
@aspnet-hello triage https://dev.azure.com/dnceng/public/_build/results?buildId=212117&view=logs

**Comment by natemcmaster** [comment]
> do we have an approved 2.1 PR in this repo that will add <PropertyGroup Condition=" '$(VersionPrefix)' == '2.1.12' "> to PatchConfig.props?

Good point. We do not, yet. I'll add that to this PR.

**Comment by aspnet-hello** [comment]
This comment was made automatically. If there is a problem contact aspnetcore-build@microsoft.com.

I've triaged the above build. I've created/commented on the following issue(s)
https://github.com/aspnet/AspNetCore-Internal/issues/2626

**Pull request merged**